pypresence/baseclient.py

```diff
@@ -13,11 +13,8 @@
 
 class BaseClient:
 
-    def __init__(self, client_id: str, **kwargs):
-        pipe = kwargs.get('pipe', 0)
-        loop = kwargs.get('loop', None)
-        handler = kwargs.get('handler', None)
-        self.isasync = kwargs.get('isasync', False)
+    def __init__(self, client_id: str, pipe=0, loop=None, handler=None, isasync=False):
+        self.is_async = isasync
```

Comment:
isasync vs is_async makes no difference really, I'm not looking to pray to PEP8.